doc/user/content/integrations/http-api.md

```diff
@@ -181,11 +181,22 @@ interface Error {
 	hint?: string;
 }
 
+interface Column {
+    name: string;
+    type_oid: number; // u32
+    type_len: number; // i16
+    type_mod: number; // i32
```

Comment:
I'm not super familiar with this doc, but it might be helpful to describe somewhere what each of these fields mean.

Reply:
+1

It looks like we're using part of the [RowDescription](https://www.postgresql.org/docs/current/protocol-message-formats.html) format, maybe link the postgres docs here?